catalog/java/io/material/catalog/bottomappbar/res/layout/cat_bottomappbar_fragment.xml

```diff
@@ -42,19 +42,22 @@
       app:layout_anchor="@id/bar"
       app:srcCompat="@drawable/ic_add_24px"/>
 
-  <FrameLayout
+  <RelativeLayout
       android:id="@+id/bottom_drawer"
       style="@style/Widget.Material3.BottomSheet"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      app:behavior_hideable="true"
```

Comment:
Revert this?

Reply:
I reverted it. But, why? Widget.Material3.BottomSheet contains app:behavior_hideable="true".

Reply:
The layout file may be re-used with other themes. Since this is not relevant to the fix, let's try our best to reduce potential breakages as much as possible.